net/asterisk/files/asterisk.init

```diff
@@ -51,7 +51,6 @@ start_service() {
   procd_open_instance
   procd_set_param command $COMMAND
   procd_append_param command \
-    -G "$NAME" \
     -U "$NAME" \
```

Comment:
In my asterisk init script I also removed the line
    -U "$NAME" \
but have set in /etc/asterisk/asterisk.conf:
```
runuser = asterisk              ; The user to run as.
rungroup = dialout              ; The group to run as.
```
and in /etc/init.d/vmmc:
```
chown :dialout /dev/vmmc$i
```
Works for me since over a year together with git master.

See also:
https://github.com/openwrt/telephony/issues/520
and
https://github.com/openwrt/openwrt/pull/3124